code/modules/meteors/meteor_spawning.dm

```diff
@@ -31,22 +31,22 @@
 	var/new_meteor = pick_weight(meteor_types)
 	new new_meteor(picked_start, picked_goal)
 
-/proc/spaceDebrisStartLoc(start_side, Z)
+/proc/spaceDebrisStartLoc(start_side, Z, distance_from_edge)
 	var/starty
 	var/startx
 	switch(start_side)
 		if(NORTH)
-			starty = world.maxy-(TRANSITIONEDGE + MAP_EDGE_PAD)
+			starty = world.maxy - (TRANSITIONEDGE + MAP_EDGE_PAD) - distance_from_edge
 			startx = rand((TRANSITIONEDGE + MAP_EDGE_PAD), world.maxx-(TRANSITIONEDGE + MAP_EDGE_PAD))
```

Comment:
any reason as to why you're only bumping up one of the coordinates instead of both?

Reply:
Yeah I forgot, fixed it.